### DISCUSSION TITLE
Show Cortana button in Windows 10 taskbar

### DISCUSSION THREAD

**Discussion by XuanChen1234** [valinet/ExplorerPatcher#2555] [Ideas]
We found that the taskbar option in ExplorerPatcher found that "Show Cortana button" was gone.
Please add that option back.
I appreciate any and all advice and help, thanks!
![257018854-265242cb-840a-4078-a828-f1735f236e21](https://github.com/valinet/ExplorerPatcher/assets/140480110/43eb213f-12e9-4ed4-95e9-c0403da001e4)

![屏幕截图 2023-12-05 155905](https://github.com/valinet/ExplorerPatcher/assets/140480110/24b31f43-e988-4153-aeb1-ceda4f233e99)

**Comment by tallman69** [comment]
Microsoft has ended support for Cortana.
https://support.microsoft.com/en-us/topic/end-of-support-for-cortana-d025b39f-ee5b-4836-a954-0ab646ee1efa
It has been replaced with Copilot.

**Comment by milestprower92** [comment]
> We found that the taskbar option in ExplorerPatcher found that "Show Cortana button" was gone. Please add that option back. ![屏幕截图 2023-12-05 155905](https://private-user-images.githubusercontent.com/140480110/287981081-24b31f43-e988-4153-aeb1-ceda4f233e99.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDIzMTc5OTAsIm5iZiI6MTcwMjMxNzY5MCwicGF0aCI6Ii8xNDA0ODAxMTAvMjg3OTgxMDgxLTI0YjMxZjQzLWU5ODgtNDE1My1hZWIxLWNlZGE0ZjIzM2U5OS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBSVdOSllBWDRDU1ZFSDUzQSUyRjIwMjMxMjExJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDIzMTIxMVQxODAxMzBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xZjg3ZDQyOTA0MzQ5N2UzY2FiNmFjODUyM2M2MGZlMTAyNGIzNzljM2EzNTA2YTFhZTg4YmNjNWU1YjRhNzU1JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.b5DE-6dNMKOc6lXhsksKaMnz9h7hhxahrWJKZ_sIaBE)

No longer possible in 22H2 and 23H2 because Microsoft has removed the code of the cortana button from the old taskbar if you want the cortana button you have to be in 21h2 (22000)

**Comment by XuanChen1234** [comment]
@valinet @Vipin2003 Please add that option back Show Cortana button in Windows 10 taskbar!!!

**Reply by elvisisvan** [reply]
is your cortana still work?
![image](https://github.com/user-attachments/assets/bdda0c23-097a-40d5-b205-c878410dc9b1)

mine doesn't
